hadoop-common-project/hadoop-common/src/site/markdown/FileSystemShell.md

```diff
@@ -415,6 +415,7 @@ Options:
 * -S: Sort output by file size.
 * -r: Reverse the sort order.
 * -u: Use access time rather than modification time for display and sorting.  
+* -e: Display the erasure coding policy of files and directories only.
```

Comment:
this is from https://issues.apache.org/jira/browse/HDFS-11647 ; unless the whole patch is to be backported, it should be left out

Reply:
I found there's a commit https://github.com/apache/hadoop/commit/668817a6cefa6025ddfe082ed71d7d317d811381#diff-eec5ebba9f19087d0813f80afdb11e40 to revert parts of this commit https://github.com/apache/hadoop/commit/fa8cd1bf28f5b81849ba351a2d7225fbc580350d#diff-26c5366750139b7b3a243b5b7a3d67bb. I'll cherry pick this revert.